deathcause.lua

```diff
@@ -0,0 +1,212 @@
+-- show death cause of a creature
+local utils = require 'utils'
+
+local DEATH_TYPES = {
```

Comment:
instead of copying this, could you make `gui/unit-info-viewer` into a [module](https://docs.dfhack.org/en/stable/docs/dev/Lua%20API.html#importing-scripts), change it's `DEATH_TYPES` table to non-local, and access it via `reqscript`? e.g. `local DEATH_TYPES = reqscript('gui/unit-info-viewer').DEATH_TYPES`

If this comes up elsewhere, we can look into moving it into a library, but no need to move it yet.

Reply:
Awesome will definitely do this. I wasn't aware of this ability